src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -71,7 +71,7 @@ public partial class Process : System.ComponentModel.Component, System.IDisposab
         public long PrivateMemorySize64 { get { throw null; } }
         public System.TimeSpan PrivilegedProcessorTime { get { throw null; } }
         public string ProcessName { get { throw null; } }
-        public System.IntPtr ProcessorAffinity { get { throw null; } set { } }
+        public System.IntPtr ProcessorAffinity { [System.Runtime.Versioning.SupportedOSPlatform("windows")] [System.Runtime.Versioning.SupportedOSPlatform("linux")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatform("windows")] [System.Runtime.Versioning.SupportedOSPlatform("linux")] set { } }
```

Comment:
The attribute is supported on properties, and the same settings are being applied here to both the getter and setter.  Can we attribute ProcessorAffinity rather than attributing both its getter and setter?